src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        int[] data = workWithFile.readFromFile(fromFileName);
```

Comment:
```suggestion
        int[] data = readFromFile(fromFileName);
```
you can access method even without object creation, since we are in the same class